docusaurus/docs/dev-docs/quick-start.md

```diff
@@ -111,87 +149,246 @@ It would help getting a bit more organized if our restaurants directory had some
 
-The restaurant is now listed in the [Collection types - Restaurant](http://localhost:1337/admin/content-manager/collectionType/api::restaurant.restaurant) view.
+</details>
+
+<details style={detailsStyle}>
+<summary style={summaryStyle}>Step 3: Add Categories</summary>
 
-#### Add Categories
+#### Step 3: Add Categories
 
-Let's go to ![Content Manager icon](/img/assets/quick-start-guide/icons/content.svg) [Content Manager > Collection types - Category](http://localhost:1337/admin/content-manager/collectionType/api::category.category) and create 2 categories:
+Let's go to ![Content Manager icon](/img/assets/quick-start-guide/icons/content.svg) _Content Manager > Collection types - Category_ and create 2 categories:
 
 1. Click on **Create new entry**.
 2. Type `French Food` in the _Name_ field.
 3. Click **Save**.
 4. Go back to _Collection types - Category_, then click again on **Create new entry**.  
 5. Type `Brunch` in the _Name_ field, then click **Save**.
 
-![GIF: Add Categories](/img/assets/quick-start-guide/qsg-handson-categories.gif)
+<ThemedImage
+alt="GIF: Add Categories"
+sources={{
+  light: '/img/assets/quick-start-guide/qsg-handson-categories.gif',
+  dark: '/img/assets/quick-start-guide/qsg-handson-categories_DARK.gif',
+}}/>
 
-The "French Food" and "Brunch" categories are now listed in the [Collection types - Category](http://localhost:1337/admin/content-manager/collectionType/api::category.category) view.
+The "French Food" and "Brunch" categories are now listed in the _Collection types - Category_ view of the ![Content Manager icon](/img/assets/quick-start-guide/icons/content.svg) _Content Manager_.
 
-#### Add a Category to a Restaurant
+Now, we will add a category to a restaurant:
 
-Go to ![Content Manager icon](/img/assets/quick-start-guide/icons/content.svg) [Content Manager > Collection types - Restaurant](http://localhost:1337/admin/content-manager/collectionType/api::restaurant.restaurant) in the navigation, and click on "Biscotte Restaurant".
+1. Go to ![Content Manager icon](/img/assets/quick-start-guide/icons/content.svg) _Content Manager > Collection types - Restaurant_ in the navigation, and click on "Biscotte Restaurant".
+2. In the **Categories** drop-down list at the bottom of the page, select "Brunch". Scroll back to the top of the page and click **Save**.
 
-In the **Categories** drop-down list, select "Brunch". Click **Save**.
+</details>
 
-### Step 3: Set Roles & Permissions
+<details style={detailsStyle}>
+<summary style={summaryStyle}>Step 4: Set Roles & Permissions</summary>
+
+### Step 4: Set Roles & Permissions
 
 We have just added a restaurant and 2 categories. We now have enough content to consume (pun intended). But first, we need to make sure that the content is publicly accessible through the API:
 
 1. Click on _General ![Settings icon](/img/assets/quick-start-guide/icons/settings.svg) Settings_ at the bottom of the main navigation.
```

Comment:
Shouldn't it be "General **>** Settings"?

Reply:
That's right! Should we keep the icon in between? In some other pages it seems we don't even mention General, we just use something like [ settings icon ] Settings. So I think I'll remove the "General >" part, what do you think? 🤔 (see https://docs.strapi.io/user-docs/settings/admin-panel for instance).

Reply:
Yes I checked too and saw that we don't mention the "General" part anywhere else, so I agree that we can remove it! "Settings" + the icon should be enough for users to find where they should go!